CONTRIBUTING.md

```diff
@@ -71,6 +71,14 @@ files for the DevTools Flutter app.
 
 To connect to your running application, paste the earlier copied observatory URL into the section "Connect to a Running App" in DevTools.
 
+To make VSCode's F5 having all DevTools` experiments enabled, add to the debugging configuration:
```

Comment:
nit: "To enable all experiments by default when you are running with VS Code, add the following to your debugging configuration:"